docs/src/oldstable/releases/upgradeFrom92.md

```diff
@@ -0,0 +1,72 @@
+# Upgrading from 0.9.2
+Upgrading the DSF from 0.9.2 to 0.9.3 involves modifying the docker-compose.yml files and recreating the containers.
+
+**As the upgrade from 0.9.0 to 0.9.1 and 0.9.1 to 0.9.2 does not require any changes except the change of the version numbers, you can directly follow the following instructions.**
+
+## Modify DSF FHIR Server Setup
+1. Preparation / Backup
+    * We recommend to create a backup of the `/opt/fhir` directory before proceeding with the upgrade.  
+    For example using: `sudo cp -rp /opt/fhir /opt/fhir_backup_pre_0.9.3_upgrade`
+
+1. Modify the DSF FHIR docker-compose.yml file, replace 0.9.2 (or 0.9.0, 0.9.1) with 0.9.3
+    ```diff
+    @@ -1,7 +1,7 @@
+     version: '3.8'
+     services:
+       proxy:
+    -    image: ghcr.io/highmed/fhir_proxy:0.9.2
+    +    image: ghcr.io/highmed/fhir_proxy:0.9.3
+         restart: on-failure
+         ports:
+           - 127.0.0.1:80:80
+    @@ -27,7 +27,7 @@ services:
+           - app
+     
+       app:
+    -    image: ghcr.io/highmed/fhir:0.9.2
+    +    image: ghcr.io/highmed/fhir:0.9.3
+         restart: on-failure
+         healthcheck:
+           test: ["CMD", "java", "-cp", "dsf_fhir.jar", "org.highmed.dsf.fhir.StatusClient"]
+    ```
+
+1. Upgrade the DSF FHIR containers  
+    From `/opt/fhir` execute  
+    ```
+    docker-compose up -d && docker-compose logs -f
+    ```
+
+## Modify DSF BPE Server Setup
+1. Preparation / Backup
+    * We recommend to create a backup of the `/opt/bpe` directory before proceeding with the upgrade.  
+    For example using: `sudo cp -rp /opt/bpe /opt/bpe_backup_pre_0.9.3_upgrade`
+
+1. Modify the DSF BPE docker-compose.yml file, replace 0.9.2 (or 0.9.0, 0.9.1) with 0.9.3
+    ```diff
+    @@ -1,7 +1,7 @@
+     version: '3.8'
+     services:
+       app:
+    -    image: ghcr.io/highmed/bpe:0.9.2
+    +    image: ghcr.io/highmed/bpe:0.9.3
+         restart: on-failure
+         healthcheck:
+           test: ["CMD", "java", "-cp", "dsf_bpe.jar", "org.highmed.dsf.bpe.StatusClient"]
+    ```
+
+1. Upgrade the DSF BPE containers  
+    From `/opt/bpe` execute  
+    ```
+    docker-compose up -d && docker-compose logs -f
+    ```
+
+1. Verify your upgrade:
```

Comment:
We always number a numbered list with "1", because markdown parsers will replace it with the correct number and we don't have to change the whole document (if we want to insert a new bullet point).